PowerEditor/src/functionList.xml

```diff
@@ -88,21 +88,21 @@
 						<nameExpr expr="\w+\Z" />
 					</className>
 					<function
-						mainExpr="^\s*((static|public|protected|private|final)*(\s+(static|public|protected|private|final))+\s+)?(function\s+)+(\w+(\s+\w+)?(\s+|\*\s+|\s+\*|\s+\*\s+))?(\w+\s*::)?(?!(if|while|for|switch))[\w~]+\s*\([^\{]*\{"
```

Comment:
We can omit this in the PHP parser, though it should be necessary for the C parsers as there is no `function` word that avoid ambiguity.